CDCgov/prime-reportstream
# React Org Dropdown

This PR introduces `GlobalContextProvider` and the org switcher for admins.

Test Steps:
1. pull `kevinhaube/add-org-dropdown`
2. `npm install` and `npm run`
3. Press F12 to open your dev tools
4. Log in
5. In dev tools, navigate to Application > Local Storage > localhost:3000 and ensure your `global-context-org` value is your normal default org
6. (Requires React Dev Tools extension) In dev tools, navigate to Components and find `GlobalContextProvider`
7. Ensure `state` shows the correct value for organization
8. Switch orgs. The page should reload. Try this on various pages!

## Changes
- `GlobalContextProvider` now wraps the pages and provides a global state
- `GlobalContext` values are persisted in `localStorage`
- Updates around the app to utilize `GlobalContext` org value

## Checklist

### Testing

### Process

## Fixes
- #2462
